app/views/home/index.html.erb

```diff
@@ -17,7 +17,7 @@
       <div class="hidden sm:mt-10 sm:flex gap-2 lg:mt-0 lg:grow lg:basis-0 lg:justify-end">
         <%= link_to "Teams", teams_path, class: "inline-flex justify-center rounded-2xl p-4 text-base font-semibold text-blue-600 hover:bg-blue-200 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %>
         <%= link_to "Login", new_user_session_path, class: "inline-flex justify-center rounded-2xl p-4 text-base font-semibold text-blue-600 hover:bg-blue-200 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %>
-        <%= link_to "Sign up", new_user_registration_path, class: "inline-flex justify-center rounded-2xl bg-blue-600 p-4 text-base font-semibold text-white hover:bg-blue-500 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %>
+        <!-- <%= link_to "Sign up", new_user_registration_path, class: "inline-flex justify-center rounded-2xl bg-blue-600 p-4 text-base font-semibold text-white hover:bg-blue-500 focus:outline-none focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-500 active:text-white/70" %> -->
```

Comment:
Protip: You can use `<%# ` erb comments instead of HTML comments here.

Reply:
I showed him that last week but guess he forgot and also old habits I suppose lol